### ISSUE TITLE
Page navigation doesn't change focus to top of form when using NVDA (WCAG 1.3.2)

### ISSUE THREAD

**Issue by lasseklovstad** [Altinn/app-frontend-react#1408]
### Description of the bug

When navigating between pages in the form while using NVDA (screen reader), focus doesn't move to the top of the form when navigating to next page with keyboard.

### Steps To Reproduce

This kan be testet in any Altinn app with Navigation button component. For Instance her: https://udir.apps.altinn.no/udir/pbu-prod-demo?survey=survey-1-aarstrinn5

1. Open NVDA (Change the Visual settings if needed, written more about in additonal info underneath)
2. tab down to "Neste Side" and click Enter (Wait for page navigation)
3. Press NVDA + Arrow up to identify current position

It starts to read header column of the Likert: "Trives ikke noe særlig" when it is excpected to read the top heading "Trivsel" instead.
![image](https://github.com/Altinn/app-frontend-react/assets/43168787/058fddc2-77f5-4b96-a245-2e9ee43b2cd3)

### Additional Information

It's easier to visualize focus when turning on this setting with NVDA:
![image](https://github.com/Altinn/app-frontend-react/assets/43168787/24d6ca3e-06c7-4979-b88a-6dfe905fde40)

[WCAG 1.3.2](https://www.uutilsynet.no/wcag-standarden/132-meningsfylt-rekkefolge-niva/91)

**Comment by Febakke** [comment]
Tested with VoiceOver and Firefox. Tested on our testapp and on Oppstartveileder for enkeltperson
I could not recreate this exactly. When tested the first element got focus either a heading or the page navigation component at the top if that was present. 

@lasseklovstad Let me know if you want me to test your service with VoiceOver to see if this is an issue with NVDA. 

I did find som weird behavior when there are an error message in the bottom and the user is still allowed to change pages. The focus would be forced to the bottom on each pageturn. This is not screen reader related is the same when using a keyboard. 

Im putting this in our WCAG backlog, we will look into it more.
